app/services/next_course_creation_step_service.rb

```diff
@@ -17,6 +17,8 @@ def execute(current_step:, current_provider:)
       :location
     when :location
       handle_location(current_provider)
+    when :accredited_body
+      :entry_requirements
```

Comment:
CodeClimate is complaining about this methods being too long. We could turn these into one-liners (`when ... then ...`) perhaps.

I've dismissed the complaint in CC for now anyway.